source/Ship.cpp

```diff
@@ -2662,9 +2666,9 @@ int Ship::Scan()
 					+ Name() + "\" completed its scan of your outfits.", Messages::Importance::High);
 	}
 
-	// Some governments are provoked when a scan is started on one of their ships.
+	// Some governments are provoked when a scan is completed on one of their ships.
 	const Government *gov = target->GetGovernment();
-	if(gov && gov->IsProvokedOnScan() && !gov->IsEnemy(government)
+	if(result && gov && gov->IsProvokedOnScan() && !gov->IsEnemy(government)
 			&& (target->Shields() < .9 || target->Hull() < .9 || !target->GetPersonality().IsForbearing())
```

Comment:
I think one of your other PRs changed what can cause a `forbearing` ship to be provoked. That should also be changed here.
I suppose you should actually open a new PR and update anywhere that `forbearing` is checked. ~~Move it to its own method.~~

Reply:
I think this is the only place where it is checked I'll check but if it is thats easy to change here

Reply:
Actually nvm this doesnt need to be changed
The new forbeiring only reacts if you do X damage and they alreadt are affected by X effect.